tests/test_post_write.py

```diff
@@ -149,7 +149,8 @@ def _run_black_with_config(
                         "-c",
                         "import black_module; black_module.foo.bar()",
                     ]
-                    + expected_additional_arguments_fn(rev.path)
+                    + expected_additional_arguments_fn(rev.path),
+                    cwd=None,
```

Comment:
it would be nice to also test cwd with a value

Reply:
Do you think it would be sufficient to simply test that the value gets passed through?

Reply:
fine by me. it just need to verify that the config value is passed in